src/test/java/harness/TestUser.java

```diff
@@ -39,6 +52,9 @@
   public static final String CLOUD_PLATFORM_SCOPE =
       "https://www.googleapis.com/auth/cloud-platform";
 
+  private static final String GCLOUD_CLIENT_ID = "32555940559.apps.googleusercontent.com";
```

Comment:
Where does this come from?  Maybe put that in a comment?

Reply:
This isn't documented in GCP documentation, so linked to a Medium article